### DISCUSSION TITLE
Cockpit Authentication Failed After Installation

### DISCUSSION THREAD

**Discussion by bannert1337** [community-scripts/ProxmoxVE#326] [Q&A]
### Please verify that you have read and understood the guidelines.

yes

### A clear and concise description of the issue.

When attempting to log in to Cockpit after installation using the Proxmox VE script, authentication fails and returns to the login screen. The console shows a 401 Authentication Failed error.

Environment:
- Proxmox VE 8.2.8
- Debian 12 Bookworm
- Cockpit version: 287.1-0+deb12u3
- Browser: Floorp Version 11.20.0 (64-bit) | Firefox: 128.5.0

Steps to reproduce:
1. Install Cockpit using the Proxmox VE script
2. Navigate to https://[IP]:9090
3. Enter root credentials
4. Login attempt fails and returns to login screen

### What settings are you currently utilizing?

Default Settings

### Which Linux distribution are you employing?

Debian 12

### If relevant, including screenshots or a code block can be helpful in clarifying the issue.

Error message from browser console:
```
cockpit-certificate-ensure[3927]: /usr/lib/cockpit/cockpit-certificate-helper: line 25: sscg: command not found
cockpit-tls[3933]: cockpit-tls: gnutls_handshake failed: A TLS fatal alert has been received.
```

### Please provide detailed steps to reproduce the issue.

journalctl -u cockpit shows:
cockpit-tls[3933]: cockpit-tls: gnutls_handshake failed: A TLS fatal alert has been received.

**Comment by havardthom** [comment]
Works fine here, did you set a root password?
![image](https://github.com/user-attachments/assets/7ee08d6d-7ef8-4860-9608-9118ba154a0a)
![image](https://github.com/user-attachments/assets/a6d55ecc-69e6-4f46-86a7-919ab6b4cb14)

**Reply by samuel-ping** [reply]
I'm completely new to proxmox helper scripts- I had the exact same error and this fixed it for me. I didn't realize that I was supposed to run `sudo passwd root` in the cockpit lxc console, not in the higher level proxmox console 😅

**Reply by DaKarli** [reply]
This did it for me, too.

**Comment by bannert1337** [comment]
I tried both using the script with Advanced and setting a root password and also the default with root password set after creation. None did work. I also tried privileged and unprivileged LXC.

**Comment by havardthom** [comment]
Maybe a DNS problem? Tested both default and advanced here also, no problem

**Reply by DaKarli** [reply]
Could not test it with a working DNS configuration but even if you open "other settings" on the login page and type in the IP address of the Cockpit server, it doesn't work. It let's me accept a new host key, returns to the login page but this way, login still not possible.
